Libs/MRML/Logic/vtkMRMLTranslator.h

```diff
@@ -0,0 +1,13 @@
+#ifndef vtkMRMLTranslator_h
+#define vtkMRMLTranslator_h
+
+class vtkMRMLTranslator
```

Comment:
```suggestion
class VTK_MRML_LOGIC_EXPORT vtkMRMLTranslator
```

Reply:
Adding `VTK_MRML_LOGIC_EXPORT` designates the class to be visible on the public API of the shared library. `VTK_MRML_LOGIC_EXPORT` is a compiler/linker-specific macro defined in an automatically generated .h file.

Reply:
Thank you, I better understand now !